pkg/component/observability/monitoring/prometheus/aggregate/prometheusrules.go

```diff
@@ -63,7 +63,7 @@ func CentralPrometheusRules() []*monitoringv1.PrometheusRule {
 								"visibility": "operator",
 							},
 							Annotations: map[string]string{
-								"description": "Node {{$labels.node}} in landscape {{$externalLabels.landscape}} was not healthy for ten scrapes in the past 30 mins.",
+								"description": "Node {{$labels.node}} in seed {{$externalLabels.seed}} was not healthy for ten scrapes in the past 30 mins.",
```

Comment:
Does `$externalLabels.seed` contain details from which landscape is the Seed? It would be best if we can display landscape + seed as you can have Seeds with the same name across different landscapes.

Reply:
No, unfortunately it does not. 

One could argue that the seed Alertmanager can be configured differently per landscape to differentiate between them, e.g., different e-mail address, different subject, etc. but I don't think that's good. I do believe we can improve here by creating a landscape label in the aggregate Prometheus as well. I spent some time in the past but I didn't find a good way for the seed to know which landscape it belongs to. But I think that could be done as a follow-up for an improvement since there are other alerts defined in the aggregate Prometheus and none of them seemed to need the `$externalLabels.landscape` so far.

What I found, though, is that this specific alert is also federated into the garden Prometheus and reimplemented there to also be sent via the garden Alertmanager channels, where a landscape label is available. I reworked commit [4757e69](https://github.com/gardener/gardener/pull/10195/commits/4757e69cb8142556813db23333f67167de3ab2c1) a bit to add seed and landscape information in the garden Prometheus alert. This way at least we have all information when the alert is sent by the garden Alertmanager. Similarly as it is done for the [SeedPodStuckInPending](https://github.com/gardener/gardener/blob/65241d8dd693b57cf69c1f90dae12c5c56371c82/pkg/component/observability/monitoring/prometheus/garden/assets/prometheusrules/seed.yaml#L256-L268) alert.

Reply:
> I reworked commit [4757e69](https://github.com/gardener/gardener/pull/10195/commits/4757e69cb8142556813db23333f67167de3ab2c1) a bit to add seed and landscape information in the garden Prometheus alert. This way at least we have all information when the alert is sent by the garden Alertmanager. Similarly as it is done for the [SeedPodStuckInPending](https://github.com/gardener/gardener/blob/65241d8dd693b57cf69c1f90dae12c5c56371c82/pkg/component/observability/monitoring/prometheus/garden/assets/prometheusrules/seed.yaml#L256-L268) alert.

Thanks!